bonds/.gitignore

```diff
@@ -0,0 +1,64 @@
+# Logs
```

Comment:
We don't need a separate .gitignore for each module - just one for the whole repo